src/inflators/model.tsx

```diff
@@ -139,6 +141,20 @@ export function inflateModel(world: HubsWorld, rootEid: number, { model }: Model
     }
   });
 
+  // TODO: Not sure if here is the best place and
+  //       this is the best way to inflate animations.
+  //       We may need to revisit.
+  if (model.animations !== undefined && model.animations.length > 0) {
+    inflateMixerAnimatable(world, rootEid, {
+      animations: model.animations
```

Comment:
I don't think we need to copy the animations off the model. This will be the thing in eid2obj for rootEid, so we can just get the animations from that in the system when we need it. Here we can just call `addComponent(world, MixerAnimatable, rootEid)`, I don't think we even need an inflator.

Reply:
I'm thinking of the opposite. Thinking if we can move animations away from Object3D to component. Object3D doesn't have animations property in the Three.js official API. It polluts the official Object3D Three.js and we need a type definition for it. And if we move animations to component, attaching/detaching animation might be easier and simpler (maybe).